integ/workflow/wf_state_options_data_attributes_loading/routers.go

```diff
@@ -0,0 +1,292 @@
+package wf_state_options_data_attributes_loading
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/integ/workflow/common"
+	"github.com/indeedeng/iwf/service"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"net/http"
+)
+
+/**
+ * This test has four states
+ *
+ * State1: Creates all Data Attributes keys that will be used in this test
```

Comment:
Can you provide details on what does WaitUntil and Execute do? F.g. State1 doesn't do anything in WaitUntil.

Reply:
Done ✅